nx/lib/nx/defn/expr.ex

```diff
@@ -643,6 +643,13 @@ defmodule Nx.Defn.Expr do
     composite({u, s, vt}, context, &expr(&1, context, :svd, [{u, s, vt}, tensor, opts]))
   end
 
+  @impl true
+  def invert(output, input) do
+    input = to_expr(input)
+    context = input.data.context
+    composite(output, context, &expr(&1, context, :invert, [output, input]))
```

Comment:
No need to call composite as it never returns a tuple:

```suggestion
    expr(output, context, :invert, [output, input])
```